db/migrate/20220830134159_create_taxes.rb

```diff
@@ -0,0 +1,13 @@
+class CreateTaxes < ActiveRecord::Migration[7.0]
+  def change
+    create_table :taxes do |t|
+      t.string :name
+      t.text :description
+      t.date :from_date
+      t.date :until_date
+      t.references :car, null: false, foreign_key: true
```

Comment:
This will need to be redone, see my comment about `has_may through` relationship